libmattermost.c

```diff
@@ -945,6 +945,56 @@ mm_fetch_url(MattermostAccount *ma, const gchar *url, const gchar *postdata, Mat
 	g_free(cookies);
 }
 
+static void
+mm_send_email_cb(PurpleBuddy *buddy)
+{
+	const gchar *email = purple_blist_node_get_string(PURPLE_BLIST_NODE(buddy),"email");
+	const gchar *first_name = purple_blist_node_get_string(PURPLE_BLIST_NODE(buddy),"first_name");
+	const gchar *last_name = purple_blist_node_get_string(PURPLE_BLIST_NODE(buddy),"last_name");
+	gchar *full_email = g_strdup("\"");
+	
+	if (first_name) {
+		full_email = g_strconcat(full_email,first_name," ",NULL);
+	}
+	if (last_name) {
+		full_email = g_strconcat(full_email,last_name," ",NULL);
+	}
+	full_email = g_strconcat(full_email,"<",email,">",NULL);
+	full_email = g_strconcat(full_email,"\"",NULL);
+
+	gchar *cmd_line = g_strdup_printf(

```

Comment:
unfortunately it requires full path for xdg-open command (which is less portable .. even if probably in most cases that will be /usr/bin/xdg-open)